synapse/federation/transport/server.py

```diff
@@ -1562,13 +1562,12 @@ def register_servlets(
                 server_name=hs.hostname,
             ).register(resource)
 
-        if hs.config.experimental.spaces_enabled:
-            FederationSpaceSummaryServlet(
-                handler=hs.get_space_summary_handler(),
-                authenticator=authenticator,
-                ratelimiter=ratelimiter,
-                server_name=hs.hostname,
-            ).register(resource)
+        FederationSpaceSummaryServlet(
+            handler=hs.get_space_summary_handler(),
```

Comment:
Note that this gets a different `handler` than the above loop. I have a branch somewhere that changes all of these endpoints to accept a `HomeServer` (like the normal C-S API endpoints) which would probably clean this all up.